mono/metadata/native-library.c

```diff
@@ -565,12 +565,16 @@ netcore_resolve_with_dll_import_resolver (MonoAssemblyLoadContext *alc, MonoAsse
 
 	HANDLE_FUNCTION_ENTER ();
 
-	MonoStringHandle scope_handle = mono_string_new_handle (domain, scope, error);
+	MonoStringHandle scope_handle;
+	scope_handle = mono_string_new_handle (domain, scope, error);
 	goto_if_nok (error, leave);
-	MonoReflectionAssemblyHandle assembly_handle = mono_assembly_get_object_handle (domain, assembly, error);
+
+	MonoReflectionAssemblyHandle assembly_handle;
+	assembly_handle = mono_assembly_get_object_handle (domain, assembly, error);
 	goto_if_nok (error, leave);
 
-	gboolean has_search_flags = flags != 0 ? TRUE : FALSE;
+	gboolean has_search_flags;
+	has_search_flags = flags != 0 ? TRUE : FALSE;
```

Comment:
`!= 0` not needed, and makes the code ambiguous to human if you don't know the precedence